extensions/file-tree-generator/src/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { Form, ActionPanel, Action } from "@raycast/api";
+import { useState } from "react";
+import { parseInput } from "./parse-input";
+import { GenerateTreeOptions, generateTree } from "./generate-tree";
+
+export default function Command() {
+  const [description, setDescription] = useState<string>("");
+  const [options, setOptions] = useState<GenerateTreeOptions>({
+    charset: "utf-8",
+    trailingDirSlash: true,
+    fullPath: false,
+    rootDot: false,
+  });
+
+  const onDropdownChange = (value: string) => {
+    if (value === "ascii" || value === "utf-8") setOptions({ ...options, charset: value });
+  };
```

Comment:
The if check is not needed, but TypeScript cannot detect that the only available options from `Form.Dropdown.Item` are `"ascii"` and `"utf-8"`